docs/FAQ/Stores.md

```diff
@@ -105,3 +105,32 @@ GetRatesAsync was called on coinaverage when the rate is outdated. It should nev
 ```
 
 The issue can be fixed by [selecting a different rate source provider](./Stores.md#how-to-change-the-exchange-rate-provider-for-invoices) in Stores > Settings > Rates, or by [updating your BTCPay Server](./ServerSettings.md#how-to-update-btcpay-server) if you're running version 1.0.3.146 or older. The update will automatically replace Coinaverage with CoinGecko.
+
+## What is a Payment Request? 
+
+You might have done specific work and want to invoice the customer in BTC. 
+Before the Payment request option, it would've been something like this; 
+
+    Customer: Can I pay the 100$ in bitcoin? 
+    You: Sure you can! Here is my BTC address, and the amount in BTC is 0.0025481 (100$ at the time I wrote the msg). 
```

Comment:
Since this is a formal doc, please use `message` instead of `msg` just to make it a bit cleaner in terms of abbreviations we use.
```suggestion
    You: Sure you can! Here is my BTC address, and the amount in BTC is 0.0025481 (100$ at the time of sending the message). 
```